mautic/mautic
# Create user with automatic credentials and email

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | features
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | https://github.com/mautic/mautic-documentation/pull/181
| Related developer documentation PR URL | N/A
| Issue(s) addressed                     | N/A




#### Description:

Added new method to create a new user.

I added a checkbox  "Generate random password and send an email with credentials to user. ".

If it is checked, inputs password will be hidden and the password will be automatically generated, 
in addition, an email with the credentials will be sent to the user.

![newusermautic](https://user-images.githubusercontent.com/50916237/109299020-ba0d5300-7834-11eb-9ee6-e02a26536a30.png)




#### Steps to test this PR:
1. Checkout to the branch of this PR
2. Generate new assets with `php bin/console mautic:assets:generate` command and clear cache with `php bin/console cache:clear` command
3. Configure email settings
4. Go to new user page
5. Check "Generate random password and send an email with credentials to user. "
6. Wait for email




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/9733"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
